src/NuGet.Core/NuGet.Build.Tasks/RestoreTask.cs

```diff
@@ -140,11 +145,16 @@ private async Task<bool> ExecuteAsync(Common.ILogger log)
 
             EmbedInBinlog = GetFilesToEmbedInBinlog(dgFile);
 
+            if (RestoreNoCache)
+            {
+                log.LogWarning(Strings.Log_RestoreNoCacheInformation);
```

Comment:
Team Review:  Do not introduce a new warning in a minor release.

Reply:
fwiw, as implemented, I don't think this warning will fail the build even with treat warnings as errors